less/bootstrap/dropdowns.less

```diff
@@ -195,6 +196,12 @@
   }
 }
 
+.caret-up {
+  .fa-caret-down::before {
+    content: @fa-var-caret-up;
+  }
+}
+
```

Comment:
Do we need to modify `dropdowns.less`? Best to leave the vendor file alone if possible.